package.json

```diff
@@ -34,15 +34,18 @@
     "concurrently": "^8.0.0",
     "cors": "^2.8.5",
     "diff2html": "^3.4.33",
+    "dotenv": "^16.4.5",
```

Comment:
@vaibssingh - can we remove this dependency as it is no longer being used? ❤️

Reply:
Sorry! This is also done.